helm/reana/templates/reana-config.yaml

```diff
@@ -34,3 +34,10 @@ data:
     {{- end }}
     quota_enabled: {{ .Values.quota.enabled }}
     file_preview_size_limit: {{ int .Values.components.reana_ui.file_preview_size_limit }}
+    launcher_repositories:
+    {{- range .Values.components.reana_ui.launcher_examples | default (list) }}
+    - name: {{ .name | quote }}
+      url: {{ .url | quote }}
+      image_url: {{ .image_url | quote }}
+      specification: {{ .specification | quote | default "reana.yaml" }}
```

Comment:
Let's remove this default here, and let's handle it in reana-ui? See also the related comment in r-ui PR